File: C/Prime.c
#include <stdio.h>
#include <math.h>
#include <stdlib.h>

int main()
{
  int number, root, x=2, r, y;
  printf("Please enter the number: ");
  scanf("%d", &number);
  root=sqrt(number);
  while (x<root) {
    r=number%x;
    y=number/x;
    if (r==0) {
      printf("The given number is not a Prime.");
      exit(0);
    }
    else {
      x=x+1;
    }
  }
  printf("The given number is a Prime.\n");
  return 0;
}

File: Python/Prime.py
import math
while True:
    Input=input("Please enter a number: ")
    try:
        float(Input)
        break
    except:
        print("That was not a number.")
        continue
x=2
y=float(Input)
root=math.sqrt(y)
r=1
while True:
    #print(x)
    #print(root)
    r=y%x
    z=y/x
    if r==0:
        print("The number is not a Prime.")
        break
    elif x>root:
        print("The number is a Prime.")
        break
    else:
        x=x+1
        continue
